### ISSUE TITLE
Multicast LOD is activating randomly / weirdly. 

### ISSUE THREAD

**Issue by ryanwilson112** [LegendsOfDota/LegendsOfDota#385]
In Custom Bot lobbies, Custom, and all_pick6 from what i've seen so far.

Fully reproducible.
No settings needed, all you need is Multicast LOD.
I suggest you turn on cheats and only level Multicast to 3.

Using the courier's abilities will activate it. Even when you're dead, if you spam any courier command it'll activate your Multicast which does nothing. Although it’s fully on cooldown as if it had actually worked for one of your skills. This works in every lobby above i've tested it.

In regards to bots, in Custom Bot, it's especially bad because it can indefinitely keep activating instead of only when you use a courier. Off cooldown, instantly activated agan. It's because of the bots doing something, I just have no idea what. I don't know if it'd act the same way with human teams in an all_pick6. I wanted to test it yesterday with Dark but we'd need more than 2 I think.

But during the bot games it seemed like a bot using the stash (buying wards, for example) activated multicast. A custom_bot game where you set radiant to only one hero doesn't activate constantly anymore. 

I also noticed multicasting Smoke Of Deceit, which seems like a bug. Will use a smoke for each multicast so a support player might lose all the smokes. 

**tl:dr:** Multicast LOD activating on courier abilities and for some unknown reason friendly Bots activate it globally. Further testing needed to see if human team-mates activate it also and why. Also remove it activating on Smoke of Deceit.

**Comment by ash47** [comment]
Pretty easy to fix, will push a fix soon.

**Comment by ash47** [comment]
Fixed the courier stuff and the dust, not 100% on the other stuff, will investigate when I get some time.

**Comment by ryanwilson112** [comment]
I just played a few bot games with multicast and it worked as intended. Didn't activate randomly anymore nor on courier / dust. 

Perhaps the courier was the root cause. I wouldn't have thought so since it was staying still in the games before whilst multicast was activating constantly, but who knows.

**Comment by ash47** [comment]
Awesome, good issue, keep up the good work!

**Comment by ash47** [comment]
this seems to be fixed.
